{
  "_annotation": {
    "cve_id": "CVE-2024-37303",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-37303.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:element-hq:synapse:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "C6336FF5-859D-56A9-AAC1-8219F2062D5C",
                "versionEndExcluding": "1.106",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          },
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:matrix:synapse:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "BFD997CF-D84F-566B-9F55-6A000B74EB7A",
                "versionEndExcluding": "1.106",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}